samples/createSubscriptionWithDeadLetterPolicy.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019-2020 Google LLC
```

Comment:
We may actually be able to just make these 2020, but it's not a huge issue.

Reply:
Done!!